src/components/storage/LoanForm.tsx

```diff
@@ -0,0 +1,134 @@
+'use client';
+
+import { Button } from '@/components/ui/Button';
+import { DatePicker } from '@/components/ui/DatePicker';
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/Form';
+import { Input } from '@/components/ui/Input';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+const formSchema = z.object({
+  name: z.string().min(1),
+  email: z.string().email(),
+  phone: z.string().min(1),
+  returnBy: z.date().min(new Date()),
+});
+
+type LoanFormParams = {
```

Comment:
I think we have used Props instead of Params throughout the project for type definitions so maybe use that everywhere for consistancy